apps/aetx/src/aetx.erl

```diff
@@ -209,6 +210,12 @@ gas_price(#aetx{ type = Type, cb = CB, tx = Tx }) when ?IS_CONTRACT_TX(Type) ->
 gas_price(#aetx{}) ->
     undefined.
 
+-spec min_gas_price(Tx :: tx(), Height :: aec_blocks:height()) -> MinGasPrice :: non_neg_integer().
+min_gas_price(AETx = #aetx{ type = Type, cb = CB, tx = Tx }, Height) when ?IS_CONTRACT_TX(Type) ->
+    min(CB:gas_price(Tx), 1 + (CB:fee(Tx) div gas_limit(AETx, Height)));
+min_gas_price(AETx = #aetx{ cb = CB, tx = Tx }, Height) ->
+    1 + (CB:fee(Tx) div gas_limit(AETx, Height)).
```

Comment:
So if fee comes up as 1000 and gas_limit as 100, then this function returns 11 rather than 10?

Reply:
Good catch, it should be
```
GasLimit = gas_limit(AETx, Height),
(CB:fee(Tx) + GasLimit - 1) div GasLimit.
```